shared/templates/auditd_lineinfile/tests/commented.fail.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+{{% if MISSING_PARAMETER_PASS %}}
+# platform = multi_platform_example
```

Comment:
I tried this approach but didn't work, because automatus expects that the elements in `MULTI_PLATFORM_MAPPING` start with `multi_platform_`. Instead introduced the "Not Applicable" platform in `FULL_NAME_TO_PRODUCT_MAPPING`